### PR TITLE
[FIX] l10n_latam_invoice_document: update of the method _compute_invoice_taxes_by_group

### PR DESCRIPTION
**Pull request by bruno-zanotti**
**Description of the issue/feature this PR addresses:**
Odoo has made some modifications to the method _compute_invoice_taxes_by_group from de Account module,
so we need to update the inherited method in the argentinian localization

**Improves:**
 - Don't depend on price_subtotal to compute amount_by_group (Odoo pr: odoo/odoo#51395)
 - Show taxes when they are zero on invoices print (Odoo pr: odoo/odoo#46981)
 - For the latam invoice report show the base property in the correct value and currency

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by vbe-odoo** [comment]
@Bruno-Zanotti  Hi!
I've tested and LGTM!
This is ready for technical review @jco-odoo 

cc' @ren-odoo

File: addons/l10n_latam_invoice_document/models/account_move.py [hunk 1]
```diff
@@ -208,25 +208,41 @@ def _compute_invoice_taxes_by_group(self):
...
+
+            # At this point we only want to keep the taxes with a zero amount since they do not
+            # generate a tax line.
+            zero_taxes = set()
+            for line in move.line_ids:
+                for tax in line.tax_ids.flatten_taxes_hierarchy():
+                    if tax.tax_group_id not in res or tax.id in zero_taxes:
+                        res.setdefault(tax.tax_group_id, {'base': 0.0, 'amount': 0.0})
+                        res[tax.tax_group_id]['base'] += tax_balance_multiplicator * (line.amount_currency if line.currency_id else line.balance)
+                        zero_taxes.add(tax.id)
```
Comment (jco-odoo) [inline addons/l10n_latam_invoice_document/models/account_move.py:236]: Should not this be tax.tax_group_id instead?

**Comment by agr-odoo** [comment]
@jco-odoo 
I was wondering if we could do another step to avoid duplicated code as it seems that most of the code of the `_compute_invoice_taxes_by_group` is the same as the original method
I mean something like https://gist.github.com/agr-odoo/bfa04ed976a460bfa608ec1e34617607

**Comment by Bruno-Zanotti** [comment]
> @jco-odoo
> I was wondering if we could do another step to avoid duplicated code as it seems that most of the code of the `_compute_invoice_taxes_by_group` is the same as the original method
> I mean something like https://gist.github.com/agr-odoo/bfa04ed976a460bfa608ec1e34617607

Thanks @agr-odoo, I've made a pr with this approach https://github.com/odoo/odoo/pull/68080

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**